internal/slicer/slicer.go

```diff
@@ -170,9 +166,18 @@ func Run(options *RunOptions) (*Report, error) {
 				if extractInfo == nil {
 					return nil
 				}
-				if _, ok := slice.Contents[extractInfo.Path]; !ok {
+				if _, ok := pathInfos[extractInfo.Path]; !ok {
 					return nil
 				}
```

Comment:
This is redundant know because we are already checking the `slice.Contents` below, right?

Reply:
I would not say so, given we are calling `addKnownPath` if the path has a wildcard. Most likely, it won't go in that block, but it's better to return earlier and be safe than sorry.

Reply:
Let's keep it here for now because when we do the merge with the Starlark PR we need to use the `pathInfo` object. The problem is that we could extracted from the loop below instead of having another map lookup, let's see how it looks with the merge.

Reply:
Alright.